core/domain/exp_services.py

```diff
@@ -316,6 +317,38 @@ def get_multiple_explorations_by_id(exp_ids, strict=True):
     return result
 
 
+def get_exploration_and_exploration_rights_by_id(exploration_id):
+    """Returns a tuple for exploration domain object and exploration rights
+        object.
+
+    Args:
+        exploration_id: str. Id of the exploration.
+
+    Returns:
+        tuple(exploration object or None, exploration rights object or None).
+    """
+    exploration_and_rights = (
+        datastore_services.fetch_multiple_entities_by_ids_and_models(
+            [
+                ('ExplorationModel', [exploration_id]),
+                ('ExplorationRightsModel', [exploration_id])
+            ]))
+    if exploration_and_rights[0][0] is not None:
+        exploration = get_exploration_from_model(
+            exploration_and_rights[0][0])
+    else:
+        exploration = None
```

Comment:
A random structural suggestion - no need to implement it - but I think this block of code might look better this way:
```
exploration = None
if exploration_and_rights[0][0] is not None:
    exploration = get_exploration_from_model(
        exploration_and_rights[0][0])
```
This is just me personally thinking lesser number of if/else blocks makes things look neat

Reply:
I agree, it looks better. Updated.